### PR TITLE
support carousel block

### PR DESCRIPTION
**Pull request by icaraps**
https://github.com/hlxsites/zeiss/issues/8

Test URLs:
- Before: https://main--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/zeiss-trumpf-und-fraunhofer-mit-deutschem-zukunftspreis-ausgezeichnet
- After: https://carousel--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/zeiss-trumpf-und-fraunhofer-mit-deutschem-zukunftspreis-ausgezeichnet

### CODE REVIEW

File: blocks/carousel/carousel.js [hunk 1]
```diff
@@ -1,51 +1,204 @@
...
+
+function template(props) {
+  return document.createRange().createContextualFragment(`
+    <div class="image-slideshow grid__container">
+      <div class="grid__structure">
+        <div class="grid__column grid__column--100 image-slideshow__wrapper">
+          <div>
+            <div class="thumbnail-slideshow thumbnail-slideshow--left-aligned">
+              
+              <!--
```
Comment (karlpauls) [inline blocks/carousel/carousel.js:11]: Is this part commented out? If so, maybe we should remove it?

**Review by karlpauls — APPROVED** [review]
Looks very good!

File: blocks/carousel/carousel.js (hunk 1 above)
Comment (icaraps) [reply blocks/carousel/carousel.js:11]: I left it in because I'm not sure what it does yet. It's worth revisiting.

Comment (icaraps) [reply blocks/carousel/carousel.js:11]: I guess I can comment it in JS style so it's not rendered yes +1

**Pull request merged**